knative-operator/deploy/resources/console_cli_download_kn_deployment.yaml

```diff
@@ -0,0 +1,141 @@
+apiVersion: v1
+kind: Namespace
+metadata:
+ name: serverless-operator
+---
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  namespace: serverless-operator
+  name: kn-cli-downloads
+spec:
+  replicas: 2
+  selector:
+    matchLabels:
+      app: serverless-operator
+      component: kn-cli-downloads
+  template:
+    metadata:
+      name: kn-cli-downloads
+      labels:
+        app: serverless-operator
+        component: kn-cli-downloads
+    spec:
+      containers:
+      - name: kn-download-server
+        terminationMessagePolicy: FallbackToLogsOnError
+        image: quay.io/openshift-knative/kn-cli-artifacts:latest
+        imagePullPolicy: Always
+        ports:
+        - containerPort: 8080
+          name: http
+          protocol: TCP
+        livenessProbe:
+          httpGet:
+            path: /
+            port: 8080
+            scheme: HTTP
+        readinessProbe:
+          failureThreshold: 3
+          httpGet:
+            path: /
+            port: 8080
+            scheme: HTTP
+        command: ["/bin/sh"]
+        resources:
+          requests:
+            cpu: 10m
+            memory: 50Mi
+        args:
+        - '-c'
+        - |
+          cat <<EOF >>/tmp/serve.py
```

Comment:
This feels like a separate file we should version control instead of an inline python script here.

Reply:
Added https://github.com/openshift/knative-client/pull/328/ to version control serve.py, and configured it to run as ENTRYPOINT for kn-cli-artifacts image.

Reply:
Removed the cmd/args from deployment manifest, now the file is versioned controlled in openshift/knative-client repo and runs part of image startup.